src/Client/DefaultRegistrationService.php

```diff
@@ -51,9 +55,15 @@ public function register(iterable $strategyHandlers): bool
             ->createRequest('POST', (string) Url::appendPath($this->configuration->getUrl(), 'client/register'))
             ->withHeader('Content-Type', 'application/json')
             ->withBody(new StringStream(json_encode([
+                // TODO: delete non-standard redundant headers
                 'appName' => $this->configuration->getAppName(),
                 'instanceId' => $this->configuration->getInstanceId(),
-                'sdkVersion' => $this->sdkName . ':' . $this->sdkVersion,
+                'sdkVersion' => 'unleash-client-php:' . $this->sdkVersion,
```

Comment:
This should use the value from `$this->sdkName`.

Reply:
In new unified headers we have `unleash-php@<semver>`, not `unleash-client-php:<semver>`. Should I change both legacy header when changing new one? I see that only way to override this value is to create Unleash object manually, with customized `registrationService`.

First 3 headers are non-standard, after introduction of `x-unleash-...`, but I'd like to keep it for now, in an unchanged state, for compatibility reasons.

Reply:
The field is intended to be overridden by downstream implementations, which might be the [Symfony SDK](https://github.com/Unleash/unleash-client-symfony) or 3rd-party clients like [the Laravel one](https://github.com/kjsoftware/laravel-unleash).

At least that was the use-case when I created it, no idea if it goes against the current policy. But IMO it makes sense that framework-specific clients can redefine the SDK name.

Reply:
New change: we are reverting to `unleash-client-<language>:<version>`